platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -215,10 +214,10 @@ The view exposes only those properties that are safe to access across project bo
 
 The example below shows how the API could be used from a `Project` configuration callback to query the root project directory in a parallel-safe way:
 
-```groovy
-gradle.lifecycle.beforeProject { project ->
-   def rootDir = project.isolated.rootProject.projectDirectory
-   println "The root project directory is $rootDir"
+```kotlin
+gradle.lifecycle.beforeProject {
+    val rootDir = project.isolated.rootProject.projectDirectory
+    println("The root project directory is $rootDir")
 }
```

Comment:
❓ Would dropping the `beforeProject` call make more sense?
The sample in full is not useful: it prints the same line as many times as there are subprojects.

One of the real benefits of the isolated view is that you can use it from a subproject's build script.
```
subproject/build.gradle.kts

checkstyle {
    val rootDir = project.isolated.rootProject.projectDirectory
    configDirectory = rootDir.dir("checkstyle")
}
```